tests/posit/connect/test_permissions.py

```diff
@@ -262,3 +265,81 @@ def test(self):
 
         # assert
         assert permission == fake_permission
+
+
+class TestPermissionsDestroy:
+    @responses.activate
+    def test_destroy(self):
+        # data
+        permission_uid = "94"
+        content_guid = "f2f37341-e21d-3d80-c698-a935ad614066"
+        fake_permissions = load_mock_list(f"v1/content/{content_guid}/permissions.json")
+        fake_followup_permissions = fake_permissions.copy()
+        fake_followup_permissions.pop(0)
+        fake_permission = load_mock_dict(
+            f"v1/content/{content_guid}/permissions/{permission_uid}.json"
+        )
+        fake_user = load_mock_dict("v1/user.json")
+        fake_group = load_mock_dict("v1/groups/6f300623-1e0c-48e6-a473-ddf630c0c0c3.json")
+
+        # behavior
+
+        # Used in internal for-loop
+        mock_permissions_get = [
+            responses.get(
+                f"https://connect.example/__api__/v1/content/{content_guid}/permissions",
+                json=fake_permissions,
+            ),
+            responses.get(
+                f"https://connect.example/__api__/v1/content/{content_guid}/permissions",
+                json=fake_followup_permissions,
+            ),
+        ]
+        # permission delete
+        mock_permission_delete = responses.delete(
+            f"https://connect.example/__api__/v1/content/{content_guid}/permissions/{permission_uid}",
+        )
+
+        # setup
+        params = ResourceParameters(requests.Session(), Url("https://connect.example/__api__"))
+        permissions = Permissions(params, content_guid=content_guid)
+
+        # (Doesn't match any permissions, but that's okay)
+        user_to_remove = User(params, **fake_user)
+        group_to_remove = Group(params, **fake_group)
+        permission_to_remove = Permission(params, **fake_permission)
+
+        # invoke
+        destroyed_permission = permissions.destroy(
+            fake_permission["principal_guid"],
+            # Make sure duplicates are dropped
+            fake_permission["principal_guid"],
+            # Extract info from User, Group, Permission
+            user_to_remove,
+            group_to_remove,
+            permission_to_remove,
+        )
+
+        # Assert bad input value
+
+        with pytest.raises(TypeError):
```

Comment:
```suggestion
        # Assert bad input value
        with pytest.raises(TypeError):
```

Is this extra whitesapce? Or is it intentional?